client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card/compact';
+import Main from 'components/main';
+import Notice from 'components/notice';
+import upgradesActions from 'lib/upgrades/actions';
+import DomainConnectAuthorizeDescription from './domain-connect-authorize-description';
+import DomainConnectAuthorizeConflicts from './domain-connect-authorize-conflicts';
+import DomainConnectAuthorizeFooter from './domain-connect-authorize-footer';
+import { actionType, noticeType } from './constants';
+
+class DomainConnectAuthorize extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
```

Comment:
I've [learned a cool trick](https://github.com/Automattic/wp-calypso/pull/14029#discussion_r117272988) lately - if all you need is to set the initial value of the state in the constructor, you can do that directly:
```
class Foo extends Component {
  state = { … }
  render() {}
}
```